### PR TITLE
Prod config for Ethos

### PR DESCRIPTION
**Pull request by jacarrete**
### Change description ###

Added Ethos prod config

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: k8s/prod/common/ethos/repl-docmosis-service.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      replicas: 2
+      memoryRequests: '512Mi'
+      cpuRequests: '250m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1500m'
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "162CD82F0363F36E81218AD218C3C19F862CD480"
```
Comment (adusumillipraveen) [inline k8s/prod/common/ethos/repl-docmosis-service.yaml]: Is that right app insights key ?

Comment (jacarrete) [reply k8s/prod/common/ethos/repl-docmosis-service.yaml]: Yes, I got it from the prod secrets. At some point it should be removed and created automatically as I don't think we are using App Insights properly.

File: k8s/prod/common/ethos/repl-docmosis-service.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      image: hmctspublic.azurecr.io/ethos/repl-docmosis-backend:prod-ec0fade0
+      ingressHost: ethos-repl-docmosis-backend-prod.service.core-compute-prod.internal
+      environment:
```
Comment (timja) [inline k8s/prod/common/ethos/repl-docmosis-service.yaml]: It looks like these 2 values need overriding for production:
https://github.com/hmcts/ethos-repl-docmosis-service/blob/master/charts/ethos-repl-docmosis-backend/values.yaml#L23-L24

Comment (jacarrete) [reply k8s/prod/common/ethos/repl-docmosis-service.yaml]: Added

**Pull request merged**